jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/model/UserTaskInstanceDesc.java

```diff
@@ -126,7 +126,7 @@ public UserTaskInstanceDesc(Long taskId, String status, String actualOwner,
         this.deploymentId = deploymentId;
         this.dueDate = dueDate;
     }
-    
+
```

Comment:
I think this files does not need to be modified, you can checkout previous version from history and commit